inference/server/oasst_inference_server/models/chat.py

```diff
@@ -39,6 +40,7 @@ class DbMessage(SQLModel, table=True):
     worker_config: inference.WorkerConfig | None = Field(None, sa_column=sa.Column(pg.JSONB))
 
     score: int = Field(0)
+    inferior_drafts: list[str] | None = Field(None, sa_column=sa.Column(pg.JSONB))
```

Comment:
I guess that the way this works is that for each prompter message with multiple drafted replies, each reply has a list of message IDs which are "inferior" to it?

Reply:
Yes, when the user selects a draft message the two other messages have their ID's added to the selected message's inferior message list.

Reply:
Ok. Thinking from the perspective of ease of querying/interpreting this data. We could probably have a `draft_evaluation` table with fields like:
```
id: str
user_id: str
selected_message_id: str
inferior_message_ids: list[str]
```

Reply:
Would it make sense to rename the `draft_evaluation` table to a more general `message_evaluation` so that it could be reused in the future for something like ChatGPT's "Better, Worse, Same" on message regeneration?

Reply:
Yeah this could make sense - will leave up to you